### ISSUE TITLE
compile error for android templates with disable_3d=yes

### ISSUE THREAD

**Issue by volzhs** [godotengine/godot#3537]
compiled with d3bd1a731c0c1d06abe15e5eea5de64453008b28 on ubuntu.

```
scons -j 4 p=android target=release disable_3d=yes colored=yes tools=no theora=no speex=no dds=no pvr=no openssl=no musepack=no disable_tooltip=yes

....

Linking Static Library ==> servers/libservers.android.opt.armv7.neon.a
Ranlib Library         ==> servers/libservers.android.opt.armv7.neon.a
Linking Static Library ==> core/libcore.android.opt.armv7.neon.a
Ranlib Library         ==> core/libcore.android.opt.armv7.neon.a
Linking Shared Library ==> bin/libgodot.android.opt.armv7.neon.so
scene/libscene.android.opt.armv7.neon.a(viewport.android.opt.armv7.neon.o):viewport.cpp:function Viewport::_camera_make_next_current(Camera*): error: undefined reference to 'Camera::make_current()'
collect2: error: ld returned 1 exit status
scons: *** [bin/libgodot.android.opt.armv7.neon.so] Error 1
scons: building terminated because of errors.
```

**Comment by freemanfromgodotengine** [comment]
Does simple `scons -j 2 p=android target=release`  work on your setup?
It worked for me few hours ago.
I use xubuntu 14.04 64-bit

**Comment by volzhs** [comment]
I tested as @freemanfromgodotengine said.
``scons -j 4 p=android target=release`` is fine.
``scons -j 4 p=android target=release disable_3d=yes`` makes compile error.

**Comment by freemanfromgodotengine** [comment]
I am trying to confirm your find... compilling in progress.

**Comment by freemanfromgodotengine** [comment]
Yes, I can confirm the bug.
`scons -j 2 p=android target=release disable_3d=yes colored=yes tools=no theora=no speex=no dds=no pvr=no openssl=no musepack=no disable_tooltip=yes`

ends with 

```
Compiling ==> core/bind/core_bind.cpp
Linking Static Library ==> core/libcore.android.opt.armv7.neon.a
Ranlib Library         ==> core/libcore.android.opt.armv7.neon.a
Linking Shared Library ==> bin/libgodot.android.opt.armv7.neon.so
scene/libscene.android.opt.armv7.neon.a(viewport.android.opt.armv7.neon.o):viewport.cpp:function Viewport::_camera_make_next_current(Camera*): error: undefined reference to 'Camera::make_current()'
collect2: error: ld returned 1 exit status
scons: *** [bin/libgodot.android.opt.armv7.neon.so] Error 1
scons: building terminated because of errors
```

**Comment by akien-mga** [comment]
Seems like a duplicate of #1701, though there is interesting log output here so I'll mention it in the other issue.

**Comment by volzhs** [comment]
I compiled successfully with ``disable_3d=yes`` before 53e237dfe4aa550fa52a1187d1b19e7b5ec555eb commit.

**Comment by akien-mga** [comment]
Oh, thanks for the info, reopening then. It might hint that #1701 might actually be misinterpreted and fixed already (or was before 53e237d), I'll have to check.

**Comment by akien-mga** [comment]
Well it seems #1701 is being used as the "can't compile with disable_3d=yes" issue, so I'll close this one as a duplicate, even if in 2016 they might have had different circumstances.
